Assets/MixedRealityToolkit/Inspectors/ExtensionTemplates/ExtensionInterfaceTemplate.txt

```diff
@@ -0,0 +1,10 @@
+using System;
+using Microsoft.MixedReality.Toolkit.Extensions;
```

Comment:
Should these files be in the project structure?
Might be better in a child folder of the assets folder or something similar.

Reply:
It does feel messy. Open to suggestions.

I tried storing templates as read-only literal strings since they're so short but that was also messy.

Reply:
I think think it'll be better to keep it outside the editor so that it doesn't take as long to import. 

Text files always seem to take the longest for whatever reason.